src/services/school/hooks/index.js

```diff
@@ -205,6 +205,18 @@ const isNotAuthenticated = async (context) => {
 	return !((context.params.headers || {}).authorization || (context.params.account && context.params.account.userId));
 };
 
+const validateSchoolNumber = (context) => {
+	if (context && context.data && context.data.officialSchoolNumber) {
+		const { officialSchoolNumber } = context.data;
+		// eg: 'BE-16593' or '16593'
+		const officialSchoolNumberFormat = RegExp('\\D{0,2}-*\\d{5}$');
+		if (!officialSchoolNumberFormat.test(officialSchoolNumber)) {
+			throw new Error(`School ID is incorrect.\n The format should be 'AB-12345' or '12345' (without the quotations) `);
+		}
```

Comment:
"School number" would be more clear instead of ID

Reply:
done